src/quemb/molbe/misc.py

```diff
@@ -432,6 +433,38 @@ def be2puffin(
         print("Running from chkfile not tested with density fitting: DF set to None")
         mf.with_df = None
```

Comment:
```suggestion
        if hasattr(mf, "with_df"): raise ValueError("Running from chkfile not tested with density fitting")
```
sorry i know this isn't part of this PR's diff, but it's probably easier to fix this as well

Reply:
great, happy to change it in this anyways!